cmd/load_test/src/main.rs

```diff
@@ -122,18 +116,21 @@ async fn claim_erc20_balances(
 ) -> eyre::Result<()> {
     let mut tasks = JoinSet::new();
 
-    for (pk, sk) in accounts {
+    for (_pk, sk) in accounts {
```

Comment:
I would leave it as the one below
```suggestion
    for (_, sk) in accounts {
```